firmware/components/driver_framebuffer/driver_framebuffer_matrix.cpp

```diff
@@ -133,7 +134,7 @@ void matrix_stack_2d_clear(matrix_stack_2d *stack) {
     stack->matrices = new matrix_2d[CONFIG_MATRIX_STACK_SIZE];
 }
 
-//returns ESP_ERROR if the stack would become too big
```

Comment:
shouldn't you use ESP_ERR_INVALID_ARG here? (and return ESP_ERR_INVALID_ARG instead of 1)

Reply:
or maybe: `returns 1 (the deprecated ESP_ERROR) if the stack would become too big`
and `return 1; // deprecated ESP_ERROR`.